src/components/GotoLineModal.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { getActiveSearch, getSelectedSource } from "../selectors";
+import actions from "../actions";
+
+import Modal from "./shared/Modal";
+
+import SearchInput from "./shared/SearchInput";
+
+import type { SourceRecord } from "../reducers/sources";
+import type { SelectSourceOptions } from "../actions/sources";
+
+type GotoLineModalState = {
+  query: ?string
+};
+
+import "./SymbolModal.css";
```

Comment:
I'm pretty sure this file is actually not being used anywhere (it's also called in `SymbolModal.js` but the `.symbol-modal-wrapper` and `.symbol-modal` classes are never applied. I might suggest merging the [remaining rules](https://github.com/devtools-html/debugger.html/blob/master/src/components/SymbolModal.css#L18-L31) into `shared/Modal.css` instead but that's just outside the scope of this pr.

Reply:
It looks the those rules are applied?

https://github.com/devtools-html/debugger.html/blob/master/src/components/SymbolModal.js#L227 `.input-wrapper`

and then the result list? https://github.com/devtools-html/debugger.html/blob/master/src/components/SymbolModal.js#L213

Reply:
Sorry, I wasn't clear, I meant that only `.symbol-modal-wrapper` and `.symbol-modal` classes don't seem to be applied and the remaining rules that I linked to might be incorporated into `Modal.css` in the process of merging the modals together more, but I'm not planning to do that on this PR.